adminapi/dataset/filters.py

```diff
@@ -137,45 +148,35 @@ def __repr__(self):
     def _serialize(self):
         return {'name': 'insidenetwork', 'networks': self.networks}
 
+
 class PublicIP(BaseFilter):
     def __repr__(self):
         return 'PublicIP()'
 
     def _serialize(self):
         return {'name': 'publicip'}
 
+
 class PrivateIP(BaseFilter):
     def __repr__(self):
         return 'PrivateIP()'
 
     def _serialize(self):
         return {'name': 'privateip'}
 
+
 class Empty(BaseFilter):
     def __repr__(self):
         return 'Empty()'
 
     def _serialize(self):
         return {'name': 'empty'}
 
+
 def _prepare_filter(filter):
     return filter if isinstance(filter, BaseFilter) else ExactMatch(filter)
 
+
 filter_classes = {
-    'exactmatch': ExactMatch,
-    'regexp': Regexp,
-    'comparison': Comparison,
-    'any': Any,
-    'any': Any,
-    'and': And,
-    'or': Or,
-    'between': Between,
-    'not': Not,
-    'startswith': Startswith,
-    'overlap': Overlap,
-    'insidenetwork': InsideNetwork,
-    'privateip': PrivateIP,
-    'publicip': PublicIP,
-    'optional': Optional,
-    'empty': Empty,
+    d.lower(): globals()[d] for d in dir() if not d.startswith('_')
```

Comment:
Should we mention that somewhere, that all classes are collected automatically? Could be dangerous in future, if one does not know about that.